### ISSUE TITLE
treeCl error with scikit-learn >= 0.20.0

### ISSUE THREAD

**Issue by robinengler** [kgori/treeCl#19]
Hi,
Just to let you know that there seems to be some incompatibility between treeCl and the latest scikit-learn module. When using scikit-learn 0.20.0 I get the error below. When using and older version, scikit-learn 0.18.0, things work fine.

Error with scikit-learn 0.20.0
```
>>> import treeCl
DEBUG:matplotlib:CACHEDIR=/home/rengler/.cache/matplotlib
DEBUG:matplotlib.font_manager:Using fontManager instance from /home/rengler/.cache/matplotlib/fontList.json
DEBUG:matplotlib.backends:backend TkAgg version unknown
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/treeCl/__init__.py", line 4, in <module>
    from .clustering import Spectral, Hierarchical, MultidimensionalScaling, Automatic, Evaluation
  File "/usr/lib64/python2.7/site-packages/treeCl/clustering.py", line 25, in <module>
    from sklearn.mixture import GMM
ImportError: cannot import name GMM
```

Apparently the problem is that sklearn.mixture.GMM was deprecated and removed in version 0.20.0
So maybe you want to add "scikit-learn < 0.20.0" to the list of requires for treeCl.

best regards,
Robin

**Comment by kgori** [comment]
Hi Robin,

Thanks for this. Are you OK using scikit-learn 0.18 for now as a workaround? I'll make a more permanent fix by updating to the new GaussianMixture class if scikit-learn is >= 0.20.

Best,
Kevin

**Comment by robinengler** [comment]
Hi Kevin,
Yes, using version 0.18 of scikit-learn is fine for me. I was just mentioning this so that you can be aware of it, and possibly fix the issue in the future (which you apparently plan to do).
cheers,
Robin

**Comment by kgori** [comment]
This issue is fixed in version 0.1.35
